ui/src/app/features/collection-manager-subject-left/collection-manager-subject-left.component.scss

```diff
@@ -0,0 +1,88 @@
+.main {
+  padding: 24px 8px 0px 8px;
+  //min-height: 425px;
+  //height: 425px;
+  //overflow-y: scroll;
+  .button {
+    padding-bottom: 16px;
+    border-bottom: 1px solid #f1f1f1;
+    .btn {
+      border: none;
+      border-radius: 4px;
+      outline: none;
+      width: 100%;
+      height: 43px;
+      color: #fff;

```

Comment:
also FYI in any CSS files, do not reference colors directly. Use values from variables.scss / colors.scss